debug.log

```diff
@@ -0,0 +1,3 @@
+[1211/080440.103:ERROR:process_reader_win.cc(151)] SuspendThread: Access is denied. (0x5)
```

Comment:
**FIX:** Please add it to `.gitignore` as this is not a project file and shouldn't be included in repository.